protobuf-sources/build.gradle.kts

```
@@ -20,7 +20,7 @@ val generateBlockNodeProtoArtifact: TaskProvider<Exec> =
         // proto/internal/unparsed.proto matches structure for
         // block-node-protobuf/block/stream/block_item.proto after update otherwise Verification
         // might fail
-        val cnTagHash = "d7cb85ac7384f809a15489635122504661131af4" // v0.65.0
+        val cnTagHash = "273aa291b7224ab4c22ff459ba666ec6fbd9930b"
```

```suggestion
        val cnTagHash = "273aa291b7224ab4c22ff459ba666ec6fbd9930b" // v0.68.2
```
BTW this means all the references should be corrected.
You should either update all the other references and the test file to 0.68.2 or go the easier route and do set this to the following so everything lines up.

```suggestion
        val cnTagHash = "96ee3efc7814668d3f0c90d7aa1e3dc174a97a35" // v0.68.0
```